apps/ledger-live-mobile/README.md

```diff
@@ -1,6 +1,13 @@
 **[We are hiring, join us! 👨‍💻👩‍💻](https://jobs.lever.co/ledger/?department=Engineering)**
 
-# ledger-live-mobile
+# Ledger Live (mobile)
+
+## Are you adding support for a blockchain to Ledger Live?
```

Comment:
I would find more natural to have this section before the `# Ledger Live` one, as it is not really about the package itself but more a redirection to somewhere else, for lost people. Then the actual doc of the package starts, once this is out the way. To me it looks closer to the "we are hiring" link above, it could be a 2nd paragraph just below.
Just my personal taste @fcipollone-ledger wdyt?

Reply:
Some people arrive on the apps/LLD (or apps/LLM) without going through ledger-live first. So keeping the message in each app is useful

Reply:
Correct :+1: I agree to have the message in both apps/LLD _and_ apps/LLM.
My suggestion was just to move the "_Are you adding support for a blockchain to Ledger Live?_" section a bit higher in the README, just below "_we are hiring_" basically.